src/domain/users/templates/editOwn.tpl.php

```diff
@@ -93,6 +93,25 @@
                                 </span>
                             </p>
 
+                            <p>
+                                <label for="messagesfrequency"><?php echo $this->__('label.messages_frequency') ?></label>
+                                <span class='field'>
+                                    <select name="messagesfrequency" class="input" id="messagesfrequency" >
+                                        <option value="">--<?php echo $this->__('label.choose_option') ?>--</option>
+                                        <option value="300" <?php if ($values['messagesfrequency'] == "300") echo " selected "; ?>>5min</option>
```

Comment:
The individual values will have to be moved to the language files for translation. 
You can add them to the bottom of the file and in here just call $this->__('label.5min')

Reply:
OK thanks a lot

Reply:
Taken in account in commit d3f9bd23ecdc3b300b6b779109eb5bf6e5d46090